README.md

```diff
@@ -49,86 +51,99 @@ on, we suggest opening an issue detailing your ideas first.
+
+#### _Yay_ is not prompting to skip packages during system upgrade.
+  The default behavior was changed after
+  [v8.918](https://github.com/Jguer/yay/releases/tag/v8.918)
+  (see [3bdb534](https://github.com/Jguer/yay/commit/3bdb5343218d99d40f8a449b887348611f6bdbfc)
+  and issue [#554](https://github.com/Jguer/yay/issues/554)).
+  To restore the package-skip behavior use `--combinedupgrade` (make
+  it permanent by appending `--save`). Note: skipping packages will leave your
+  system in a
+  [partially-upgraded state](https://wiki.archlinux.org/index.php/System_maintenance#Partial_upgrades_are_unsupported).
+
+#### Sometimes diffs are printed to the terminal, and other times they are paged via _less_. How do I fix this?
+  _Yay_ uses `git diff` to display diffs, which by default tells _less_ not to
+  page if the output can fit into one terminal length. This behavior can be
+  overridden by exporting your own flags (`export LESS=SRX`).
+
+#### _Yay_ is not asking me to edit PKGBUILDS, and I don't like the diff menu! What can I do?
   `yay --editmenu --nodiffmenu --save`
 
-#### Only act on AUR packages or only on repo packages?
-  `yay -{OPERATION} --aur`
+#### How can I tell _yay_ to act only on AUR packages, or only on repo packages?
+  `yay -{OPERATION} --aur`  
   `yay -{OPERATION} --repo`
 
-#### `Out Of Date AUR Packages` message is displayed, why doesn't `yay` update them?
-  This means the package has been flagged out of date on the AUR, but its
-  maintainer has not yet updated the `PKGBUILD`.
+#### An `Out Of Date AUR Packages` message is displayed. Why doesn't _yay_ update them?
+  This message does not mean that updated AUR packages are available. It means
+  means the packages have been flagged out of date on the AUR, but
+  their maintainers have not yet updated the `PKGBUILD`s
+  (see [outdated AUR packages](https://wiki.archlinux.org/index.php/Arch_User_Repository#Foo_in_the_AUR_is_outdated.3B_what_should_I_do.3F)).
 
-#### Yay doesn't install dependencies added to PKGBUILD during installation.
-  Yay resolves all dependencies ahead of time. You are free to edit the
+#### _Yay_ doesn't install dependencies added to a PKGBUILD during installation.
+  _Yay_ resolves all dependencies ahead of time. You are free to edit the
   PKGBUILD in any way, but any problems you cause are your own and should not be
   reported unless they can be reproduced with the original PKGBUILD.
 
 ## Examples of Custom Operations
 
-* `yay <Search Term>` presents package selection menu
-* `yay -Ps` prints system statistics
-* `yay -Pu` prints update list
-* `yay -Yc` cleans unneeded dependencies
-* `yay -G` downloads PKGBUILD from ABS or AUR
-* `yay -Y --gendb` generates development package DB used for devel updates.
-* `yay -Syu --devel --timeupdate` Normal update but also check for development
-  package updates and uses PKGBUILD modification time and not version to
-  determine update
+* present package-installation selection menu  
```

Comment:
This way makes it so that at a glance it's hard to tell if the command is part of the text above or below. The man page has it right, maybe you could make it like that.

Reply:
OK, formatted this section like the man page. I think it looks better this way too.